src/main.ts

```diff
@@ -24,7 +24,13 @@ export async function run() {
       let token = core.getInput('token');
       let auth = !token || isGhes() ? undefined : `token ${token}`;
 
-      const installDir = await installer.getGo(versionSpec, stable, auth);
+      const checkLatest = core.getBooleanInput('check-latest');
```

Comment:
If the input isn't set, will this be `false` or `undefined`?

Reply:
It will be false.